README.md

```diff
@@ -53,7 +53,9 @@ $ yarn add webpack-sentry-plugin --dev
 
 - `organization`: **Required**, Sentry organization to upload files to
 
-- `project`: **Required**, Sentry project to upload files to
+- `project`: **Required**, Sentry project to upload files to. (Only one of `project` and `projects` is required; `projects` takes precedence if both are provided.)
+
+- `projects`: **Required**, Array of sentry projects to upload files to. (Only one of `project` and `projects` is required; `projects` takes precedence if both are provided.)
```

Comment:
I'm just bikeshedding so take with a grain of salt: It might be simpler to provide a single `project` argument that accepts a single string or an array of strings. This eliminates the slightly complicated "required"/precedence explanation in the docs, and the possibility of providing both `project` or `projects`.